### ISSUE TITLE
Import of Data.Text.Lazy.Binary

### ISSUE THREAD

**Issue by ddssff** [robstewart57/rdf4h#28]
In `Data.RDF.Types` there is an import of instances

    import Data.Text.Lazy.Binary ()

however, I cannot find a package that has or ever had such a module.  I changed it to import `Data.Text.Binary ()` from text-binary and that seems to work, but I am puzzled.

**Comment by cordawyn** [comment]
Probably somewhat related to this issue is that I can no longer compile the project even at its latest head: c2297f25054cbf768dafd2f9922709e5e289e27e. Only after I've removed all lines with `instance Binary *` declarations, I managed to compile it. (It seems `import Data.Binary` in `src/Data/RDF/TriplesGraph.hs` doesn't do much as well). How did this "Binary" thing creep into our cute little project? ;-)
It appears, @robstewart57 enabled it for RDF graph serialisation in 2773e4c10c5d386a5ae30165f6ebf541130c41cd... Is it still valid, @robstewart57 ?

**Comment by robstewart57** [comment]
Hi @cordawyn .

> Probably somewhat related to this issue is that I can no longer compile the project even at its latest head:

What compiler errors relating to the Binary instances are you getting? I'll take a look into it.

>  How did this "Binary" thing creep into our cute little project?

The reason for adding Binary instances is to allow users to serialise RDF graphs, for example should they want to persist a graph as a Haskell bytestring to file, or if they wish to transmit an RDF graph between computers over a network with one of the distributed Haskell libraries.

**Comment by cordawyn** [comment]
Hi @robstewart57! Long time no see :)
Here's the output of `make`:
```
Building rdf4h-1.3.2...
Preprocessing library rdf4h-1.3.2...
[ 1 of 14] Compiling Data.RDF.Types   ( src/Data/RDF/Types.hs, dist/build/Data/RDF/Types.o )

src/Data/RDF/Types.hs:514:10:
    No instance for (Binary T.Text)
      arising from a use of `binary-0.7.2.3:Data.Binary.Class.$gdmput'
    Possible fix: add an instance declaration for (Binary T.Text)
    In the expression: (binary-0.7.2.3:Data.Binary.Class.$gdmput)
    In an equation for `put':
        put = (binary-0.7.2.3:Data.Binary.Class.$gdmput)
    In the instance declaration for `Binary PrefixMappings'
make: *** [build] Error 1
```
(I'm still using good old ghc-7.6.3, hopefully it's not the issue though).

**Comment by robstewart57** [comment]
OK @cordawyn I've pushed a commit that bumps the lower bounds for text to 1.2.1.0 , which is when the Binary instances were added to that library. See https://github.com/robstewart57/rdf4h/commit/009d7e7937dc1c8fb9543cb93ebee61a3bbb5474 , which is now HEAD.

**Comment by cordawyn** [comment]
@robstewart57 That worked for me, thanks!

**Comment by robstewart57** [comment]
Fixed in 0ffc85bfbd31aaad681f8a30fc11cb3909f05373 .
